net.pdl2ork.PdL2Ork.yml

```diff
@@ -0,0 +1,197 @@
+id: net.pdl2ork.PdL2Ork
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+default-branch: stable
+command: pd-wrapper.sh
+finish-args:
+# Display
+  - --share=ipc
+  - --socket=x11
+  #- --socket=wayland
+  - --device=dri
+# Audio
+  - --socket=pulseaudio
+  # currently disabled per a flathub maintainer's request
+  #- --system-talk-name=org.freedesktop.RealtimeKit1
+  - --filesystem=xdg-run/pipewire-0
+# Filesystem access (pd doesn't yet work with portals)
+  - --filesystem=home
+# MIDI access (Flatpak doesn't yet support exposing only MIDI devices)
+  - --device=all
+# Linux Audio plugins
+  - --env=LADSPA_PATH=/app/extensions/Plugins/ladspa:/app/lib/ladspa
+add-extensions:
+  org.freedesktop.LinuxAudio.Plugins:
+    directory: extensions/Plugins
+    version: '23.08'
+    add-ld-path: lib
+    merge-dirs: ladspa
+    subdirectories: true
+    no-autodownload: true
+
+cleanup:
```

Comment:
Add cleanup:


- /etc - libtirpc generates some config files here, are they needed?
- /lib/pkgconfig
- '*.a*
- '*.la'
- /share/info
- /share/man

Reply:
I have added these to the cleanup part. I don't think I need any of them. Will update them with next commit.